plotly/presentation_objs/presentation_objs.py

```diff
@@ -0,0 +1,1199 @@
+                                 'lineHeight': 'normal',
+                                 'minWidth': 20,
+                                 'opacity': 1,
+                                 'textAlign': 'center',
+                                 'textDecoration': 'none'},
+                     'Heading 1': {'color': '#3d3d3d',
+                                   'fontFamily': 'Open Sans',
+                                   'fontSize': 26,
+                                   'fontStyle': 'normal',
+                                   'fontWeight': 400,
+                                   'lineHeight': 'normal',
+                                   'minWidth': 20,
+                                   'opacity': 1,
+                                   'textAlign': 'center',
+                                   'textDecoration': 'none'},
+                     'Heading 2': {'color': '#3d3d3d',
+                                   'fontFamily': 'Open Sans',
+                                   'fontSize': 20,
+                                   'fontStyle': 'normal',
+                                   'fontWeight': 400,
+                                   'lineHeight': 'normal',
+                                   'minWidth': 20,
+                                   'opacity': 1,
+                                   'textAlign': 'center',
+                                   'textDecoration': 'none'},
+                     'Heading 3': {'color': '#3d3d3d',
+                                   'fontFamily': 'Open Sans',
+                                   'fontSize': 11,
+                                   'fontStyle': 'normal',
+                                   'fontWeight': 700,
+                                   'lineHeight': 'normal',
+                                   'minWidth': 20,
+                                   'opacity': 1,
+                                   'textAlign': 'center',
+                                   'textDecoration': 'none'}}
+
+
+def _empty_slide(transition, id):
+    empty_slide = {'children': [],
+                   'id': id,
+                   'props': {'style': {}, 'transition': transition}}
+    return empty_slide
+
+
+def _box(boxtype, text_or_url, left, top, height, width, id, props_attr,
+         style_attr, paragraphStyle):
+    children_list = []
+    fontFamily = "Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace"
+    if boxtype == 'Text':
+        children_list = text_or_url.split('\n')
+
+        props = {
+            'isQuote': False,
+            'listType': None,
+            'paragraphStyle': paragraphStyle,
+            'size': 4,
+            'style': copy.deepcopy(_paragraph_styles[paragraphStyle])
+        }
+
+        props['style'].update(
+            {'height': height,
+             'left': left,
+             'top': top,
+             'width': width}
+        )
+
+    elif boxtype == 'Image':
+        props = {
+            'height': 512,
```

Comment:
could you comment where `512` came from?

Reply:
How's this:
```
# the default height, width for an Image are
# both set to 512
```